src/components/Container.js

```diff
@@ -19,116 +23,135 @@ class Container extends Component {
-                            (modal.validStatus &&
-                                !modal.validStatus.initialValue)
-                        }
-                     />
-                 }
 
-                 {rawModal.visible && (
-                     <RawModal
-                         modalTitle={modalTitle}
-                         modalDescription={modalDescription}
-                         windowType={rawModal.type}
-                         viewId={rawModal.viewId}
-                     >
-                        <div className="document-lists-wrapper">
-                             <DocumentList
-                                 type="grid"
-                                 windowType={rawModal.type}
-                                 defaultViewId={rawModal.viewId}
-                                 selected={selected}
-                                 selectedWindowType={selectedWindowType}
-                                 setModalTitle={setModalTitle}
-                                 setModalDescription={setModalDescription}
-                                 fetchQuickActionsOnInit={!(includedView.windowType && includedView.viewId)}
-                                 modalDescription={this.modalDescription}
-                                 isModal={true}
-                                 processStatus={processStatus}
-                                 includedView={includedView}
-                                 inBackground={
-                                     includedView.windowType &&
+                    {modal.visible && (
+                        <Modal
+                            windowType={modal.type}
+                            dataId={modal.dataId ? modal.dataId : dataId}
+                            data={modal.data}
+                            layout={modal.layout}
+                            rowData={modal.rowData}
+                            tabId={modal.tabId}
+                            rowId={modal.rowId}
+                            modalTitle={modal.title}
+                            modalType={modal.modalType}
+                            modalViewId={modal.viewId}
+                            isAdvanced={modal.isAdvanced}
+                            relativeType={windowType}
+                            relativeDataId={dataId}
+                            triggerField={modal.triggerField}
+                            query={query}
+                            viewId={query && query.viewId}
+                            rawModalVisible={rawModal.visible}
+                            indicator={indicator}
+                            modalViewDocumentIds={modal.viewDocumentIds}
+                            closeCallback={closeModalCallback}
+                            modalSaveStatus={
+                                modal.saveStatus &&
+                                modal.saveStatus.saved !== undefined ?
+                                    modal.saveStatus.saved : true
```

Comment:
not sure if is related, but the "saveStatus" (same as validStatus) are provided only when it's changed.
So it might be that saveStatus is missing from API response, just because nothing changed.

Reply:
That's most likely the correct explanation, thanks